validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java

```diff
@@ -243,6 +245,17 @@ public static ValidatorClientService create(
                   .propagateTo(validatorClientService.initializationComplete);
               return SafeFuture.COMPLETE;
             })
+        .exceptionally(
+            er -> {
+              Optional<Throwable> maybeCause =
+                  ExceptionUtil.getCause(er, InvalidConfigurationException.class);
+              if (maybeCause.isPresent()) {
+                LOG.warn(maybeCause.get().getMessage());
+              } else {
+                LOG.error("Error was encountered during validator client service start up.", er);
+              }
+              return null;
+            })
         .ifExceptionGetsHereRaiseABug();
```

Comment:
this cant stay here iirc, we need to remove `ifExceptionGetsHereRaiseABug` - we've handled the error before that...

Reply:
Yes just working that one out. The compiler complains if we don't check the result of the `exceptionally()`.

Reply:
can probably just do a .always wtih log.trace 'finished starting validator client service'